src/trace/span-inferrer.ts

```diff
@@ -55,7 +55,7 @@ export class SpanInferrer {
   ): SpanWrapper {
     const options: SpanOptions = {};
     const domain = event.requestContext.domainName;
-    const path = event.rawPath || event.requestContext.routeKey;
+    const path = event.rawPath || event.requestContext.path || event.requestContext.routeKey;
```

Comment:
@astuyve I'm afraid that the right change here should be:

```
const path = event.rawPath || event.requestContext.resourcePath || event.requestContext.routeKey;
```

Reply:
Hmm, I see. In our tests they were equivalent, is that not the case in your app? Can you show me the difference in your payload?
https://github.com/DataDog/datadog-lambda-js/blob/main/event_samples/api-gateway-v1.json#L4